elixir/apps/web/test/support/acceptance_case/vault.ex

```diff
@@ -89,8 +89,7 @@ defmodule Web.AcceptanceCase.Vault do
 
     :ok =
       request(:put, "identity/oidc/client/firezone", %{
-        redirect_uris:
-          "#{endpoint_url}/#{account.id}/sign_in/providers/#{provider.id}/handle_callback"
+        redirect_uris: "#{endpoint_url}/#{account.id}/providers/#{provider.id}/handle_callback"
```

Comment:
I think we should keep `sign_in` prefix for internal routes, otherwise, we will end up with name collisions again. (Eg. if we will need `providers` for some other entity.) Maybe instead of removing landing page and changing all the routes we just make landing do a permanent redirect at all times?

Reply:
Fixed!